src/interactive/authoring.tsx

```diff
@@ -20,31 +23,144 @@ export const AuthoringComponent: React.FC<Props> = ({initMessage}) => {
     });
   };
 
-  const handleFakeSensor = (e: React.ChangeEvent<HTMLInputElement>) => updateAuthoredState({useFakeSensor: e.target.checked});
+  const handleFakeSensor = (e: React.ChangeEvent<HTMLInputElement>) => {
+    updateAuthoredState({useFakeSensor: e.target.checked, sensorUnit: ''});
+  };
+
+
+  const handleUnitChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    updateAuthoredState({sensorUnit: e.target.value});
+  };
+
+
+  const handleRecordedDataChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    const value = e.target.value;
+
+    if(authoredState?.sensorUnit) {
+      let data = [] as number[][];
+      let rows = [];
+      try {
+        rows = value.split('\n');
+        for (let row of rows) {
+          data.push(row.split(/,/g).map(d => parseFloat(d)));
+        }
+      }
+      catch (e) {
+        console.error('cant parse data', e);
```

Comment:
I'm wondering if we should show some UI element to let the authors know the data can't be parsed?